File: csharp/csharp/Examples/03_output_calibration.cs
namespace EAProject.Examples
{
    // Calibrates output channel and save the new sensitivty to "Output Channels.xml"
    class Output_calibration
    {
        public static void Main()
        {
            var engine = new EAProject.EA_Engine();
            engine.SetOutputChannel(1, "3670-A", true);
            engine.RunOutputCalibration(1, 1, frequency: 1000, level: 0.01, duration: 10);
        }
    }
}

File: python/Examples/03_output_calibration.py
from HelpFunctions.Engine import EA_Engine

# Calibrates output channel and save the new sensitivity to "Output Channels.xml"
if __name__ == "__main__":
    engine = EA_Engine()
    engine.setOutputChannel(1, "3670-A", True)
    engine.runOutputCalibration(1, 1)